src/main/java/racingcar/controller/RaceController.java

```diff
@@ -0,0 +1,73 @@
+package racingcar.controller;
+
+import racingcar.domain.model.*;
+import racingcar.domain.model.dto.FinalWinnerDTO;
+import racingcar.domain.model.dto.RoundResultDTO;
+import racingcar.global.ErrorMessage;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RaceController implements Controller {
+    public static final int START_ROUND = 1;
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RaceController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    @Override
+    public void execute() {
+        RacingCars racingCars = inputName();
+        Race race = raceProcess(racingCars);
+        printResult(race);
+    }
+
+    private void printResult(Race race) {
+        String finalResult = FinalWinnerDTO.from(race).getWinnersAsString();
+        outputView.printWinningResult(finalResult);
+    }
+
+    private Race raceProcess(RacingCars racingCars) {
+        outputView.printTryNumberNotice();
+        TryNumber tryNumber = validateInputTryNumber(inputView.inputTryNumber());
```

Comment:
시도 횟수이므로 
```suggestion
        outputView.printTryCountNotice();
        TryCount tryCount = validateInputTryCount(inputView.inputTryCount());
```
이 네이밍은 어떤가요?